server/src/controllers/app/app.controller.ts

```diff
@@ -0,0 +1,16 @@
+import { Controller, Get } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { AppServicesService } from "../../services/use-cases/app/app-services.service";
```

Comment:
Когда у нас уже несколько ../.. то это чаще всего уже не ок, лучше тогда сверху проекта идти /server/services etc

Reply:
Have a problems with deploy on Vercel with Alias path config, like it's not a problem to fix it and it will be < but in priority to deploy and other your edits